checksec

```diff
@@ -1752,12 +1752,17 @@ chk_proc_libs () {
 }
 
 chk_fortify_file () {
+  # if first char of pathname is '~' replace it with '${HOME}'
+  if [[ "${CHK_FORTIFY_FILE:0:1}" = '~' ]] ; then
+    CHK_FORTIFY_FILE=${HOME}/${CHK_FORTIFY_FILE:1}
+  fi
+
   if [[ -z "${CHK_FORTIFY_FILE}" ]] ; then
     printf "\033[31mError: Please provide a valid file.\033[m\n\n"
   exit 1
   fi
   # does the file exist?
-  if [[ ! -e "${CHK_FORTIFY_FILE}" ]] ; then
+  if [[ ! -f "${CHK_FORTIFY_FILE}" ]] ; then
```

Comment:
-f checks for a regular file only; -e for *any* file type (dir/slink/dev node/etc), which is not what we want here